forge-gui-desktop/src/main/java/forge/GuiDesktop.java

```diff
@@ -144,15 +144,22 @@ public ISkinImage getCardArt(final PaperCard card, final boolean backFace) {
     }
 
     @Override
-    public ISkinImage createLayeredImage(final FSkinProp background, final String overlayFilename, final float opacity) {
+    public ISkinImage createLayeredImage(final PaperCard paperCard, final FSkinProp background, final String overlayFilename, final float opacity) {
         final BufferedImage image = new BufferedImage(background.getWidth(), background.getHeight(), BufferedImage.TYPE_INT_ARGB);
         final Graphics2D g = image.createGraphics();
         final FSkin.SkinImage backgroundImage = FSkin.getImage(background);
         FSkin.drawImage(g, backgroundImage, 0, 0, background.getWidth(), background.getHeight());
+        final int cardImageWidth = 90;
+        final int cardImageHeight = 128;
+        BufferedImage cardImage = null;
+        if (paperCard != null)
+            cardImage = ImageCache.scaleImage(paperCard.getCardImageKey(), cardImageWidth, cardImageHeight, false, null);
```

Comment:
nice work

I think you could just move the `scaleImage` after the `doesFileExist` for performance to avoid a potential image operation that might not be used?

Reply:
done